src/routes/+page.svelte

```diff
@@ -17,4 +37,56 @@
+        <a href="/"><Logo class="md:w-[2vw]" /></a>
+        <a href="/" class="absolute left-1/2 -translate-x-1/2"><CoreProject /></a>
+        <div class="flex items-center md:gap-[1vw]">
+            <a
+                href="/user/register"
+                class="btn hidden bg-surface-400 font-semibold leading-none md:flex md:rounded-[0.5vw] md:px-[1.25vw] md:py-[0.85vw] md:text-[1vw]"
+            >
+                Register
+            </a>
+            <a
+                href="/user/login"
+                class="btn rounded-[0.4rem] bg-primary-500 px-5 py-4 text-base font-semibold leading-none md:rounded-[0.5vw] md:px-[1.25vw] md:py-[0.85vw] md:text-[1vw]"
+            >
+                Login
+            </a>
+        </div>
+    </navbar>
+    <landing class="flex items-center justify-between h-screen md:px-[10vw]">
+        <content>
+            <div class="flex flex-col md:gap-[2vw] relative">
+                <div class="flex flex-col md:text-[5vw] font-bold md:leading-[4.5vw]">
+                    <span>Unlock the World of</span>
+                    <typewriter-content>
+                        <typewriter bind:this={typewriter_el} />
+                    </typewriter-content>
+                </div>
+                <div class="flex flex-col md:gap-[1vw]">
+                    <div class="w-max flex flex-col md:gap-[0.25vw] leading-none">
+                        <span class="md:text-[1.5vw] font-semibold text-surface-50">Welcome to CoreProject</span>
+                        <gradient-border class="flex w-full md:h-[0.2vw] bg-gradient-to-r from-primary-500 to-error-500" />
+                    </div>
+                    <cta-buttons class="flex items-center md:gap-[1vw] w-max">
+                        <button
+                            class="btn rounded-[0.4rem] bg-gradient-to-r from-primary-500 to-error-500 px-5 py-4 text-base font-semibold leading-none md:rounded-[0.5vw] md:px-[1.25vw] md:py-[1vw] md:text-[1vw] shadow-2xl shadow-primary-800"
+                        >
+                            <span>Get Started</span>
+                            <Chevron class="md:w-[1.25vw] -rotate-90" />
+                        </button>
+                    </cta-buttons>
+                </div>
+            </div>
+        </content>
+        <mascot class="relative flex flex-shrink-0 md:h-[40vw] self-end">
+            <img src="images/characters/hibiki/hibiki.png" alt="Hibiki" class="z-10 h-full" />
+            <!-- gradients -->
+            <gradient class="absolute left-1/4 md:w-[15vw] md:h-[20vw] rotate-45 bg-primary-900 md:rounded-[2vw] md:blur-[3.5vw]" />
+            <gradient class="absolute md:bottom-[2.5vw] right-0 md:w-[15vw] md:h-[20vw] rotate-45 bg-primary-800 md:rounded-[2vw] md:blur-[4vw]" />
+            <gradient class="absolute md:bottom-[1vw] left-1/4 md:w-[10vw] md:h-[20vw] rotate-45 bg-surface-400 md:rounded-[2vw] md:blur-[3.5vw]" />
```

Comment:
Everything should be absolute

Reply:
Everything? that `img` too?

Reply:
Yes

Reply:
Uhh I think we dont need this since we added `self-end`.
absolute isn't going to make any changes